src/components/project-overview/project-overview.tsx

```diff
@@ -1,29 +1,60 @@
 import { Project as ProjectInterface } from '@/app/dataModels/project';
 import Project from '@/components/project-card/project-card';
 import { getLocalProjectData } from '@/utils/get-local-project-data/get-local-project-data';
-import { Box, Card } from '@mui/material';
+import { Box, Button, Container, Typography } from '@mui/material';
+import { Lato } from 'next/font/google';
 import { Fragment } from 'react';
 import './project-overview.scss';
 
+const lato = Lato({
```

Comment:
if this is something global, let's put it into a global space.

Reply:
Addressed in [297792e](https://github.com/OpenBCca/openbc-web/pull/92/commits/297792e8aa6e108354a56e0cebfe14e20f12f1bf#diff-29d0a55f5ca1a9951042587f8b891bb4cd8b204f5bc855dc2468ed44cde2f5b3R8-R11)